portfolio/src/main/java/com/google/sps/data/UserLoginInfo.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+public final class UserLoginInfo {
+
+  private final boolean loginStatus;
+  private final String redirectUrl;
+
+  public UserLoginInfo(Boolean loginStatus, String redirectUrl) {
```

Comment:
does it need to be Boolean here too?
also exactly what is loginStatus? what does it mean when loginStatus is true? if it indicates if the user is logged in, it might be better to rename the property as isLoggedIn